widgets/common/src/hooks/useLabelInput.js

```diff
@@ -29,7 +29,7 @@ function useLabelInput(onChange, { allowKnownOnly = false, initialValue = '' })
         resetInput: () => {
             resetInputValue();
             onChange(initialValue);
-            if (!allowKnownOnly) unsetInvalidCharacterTyped();
+            if (!allowAdditions) unsetInvalidCharacterTyped();
```

Comment:
Optional:

How about unsetting the flag anyway?

```suggestion
            unsetInvalidCharacterTyped();
```

The behavior should be the same as now, and the code gets simpler :slightly_smiling_face:

Reply:
What your suggesting was already part of this PR. I was debating with myself, but as it is very minor I'm leaving it for now.